src/components/core/ContextSelector/index.tsx

```diff
@@ -61,16 +67,16 @@ const ContextSelector: React.FC = () => {
             const selectedItem = React.useMemo(() => {
                 const mergedItems = mergeDropdownSectionItems(dropdownSections);
                 const selectedItem = mergedItems.find(option => option.isSelected === true);
-                return selectedItem;
+                return selectedItem as SearchableContextDropdownOption;
             }, [dropdownSections]);
 
             const selectedValue = React.useMemo(() => {
                 if (isOpen) {
                     return queryText;
                 } else if (selectedItem) {
-                    return selectedItem.title;
+                    return `${selectedItem.title} (${selectedItem.contextType.id})`;
                 } else if (currentContext) {
-                    return currentContext.title;
+                    return `${currentContext.title} (${currentContext.type.id})`;

```

Comment:
Should we use the function you made to map context type to "human readable"? E.g. `ProjectMaster `-> `Project master`